src/containers/Bookings.js

```diff
@@ -1,19 +1,49 @@
 import React, { Component } from "react";
 import Search from "../components/Search.js";
+import Results from "../components/Results.js";
+
 // import Results from "../components/Results.js";
-// import * as FakeBookings from "../data/fakeBookings.json";
+import * as FakeBookings from "../data/fakeBookings.json";
+
+
 
 export default class Bookings extends Component {
-  search = () => {
-    console.info("to do!");
-  };
+  constructor(props) {
+    super(props);
+    this.state = { searched: false };
+    this.bookingData = null;
```

Comment:
Although as you have spotted you can do it this way, per the React pattern this data belongs to the state, i.e.
`
this.state = { 
  searched: false,
  bookingData: null
 };
`